git-open

```diff
@@ -56,12 +56,51 @@ if [[ -z "$giturl" ]]; then
   exit 1
 fi
 
+ssh_config=${ssh_config:-~/.ssh/config}
```

Comment:
Think these need to be quoted just in case the user's home dir has a space in it

Reply:
`shellcheck` didn't catch this one :smile:. I'll make the fix.